students/shawn_degraw/session03/mailroom.py

```diff
@@ -0,0 +1,100 @@
+#Mailroom Part 1
+import sys
+
+#donor_db definition and default values
+donor_db = [("John Smith", [500.00, 150.00, 20.00]),
+            ("Jane Doe", [340.00, 30.00, 200.00]),
+            ("Jason Bourne", [240.00, 140.00]),
+            ("John Wick", [1000.00]),
+            ("GI Jane", [150.00, 60.00])
+           ]
+
+#Thank you letter template
+thankyouletter = "\n".join(("","Dear {name},","","Thank you for your generous donation of ${amount:.2f} to our cause.","Your donations help keep Python great!","","Sincerely","","The Python Project",""))
```

Comment:
Minor: Consider breaking to new lines after each comma, or you may want to use a `'''...'''` string for a multi-line string like this. Just to increase the code readability.